airflow/providers/ssh/hooks/ssh.py

```diff
@@ -63,13 +63,13 @@ class SSHHook(BaseHook):
     :type keepalive_interval: int
     """
 
-    # key type name to paramiko PKey class
-    _default_pkey_mappings = {
-        'dsa': paramiko.DSSKey,
-        'ecdsa': paramiko.ECDSAKey,
-        'ed25519': paramiko.Ed25519Key,
-        'rsa': paramiko.RSAKey,
-    }
+    # List of classes to try loading private keys as, ordered (roughly) by most common to least common
+    _pkey_loaders = (
```

Comment:
We were never using the keys of this dict, so I changed it to just a list anyway.